modules/abi-oci-resources-services.adoc

```diff
@@ -18,7 +18,7 @@ To ensure compatibility with {product-title}, you must set `A` as the record typ
 * `api-int.<cluster_name>.<base_domain>`, which targets the `apiVIP` parameter of the API load balancer.
 * `*.apps.<cluster_name>.<base_domain>`, which targets the `ingressVIP` parameter of the Ingress load balancer.
 
-The `api.+*+` and `api-int.+*+` DNS records relate to control plane machines, so you must ensure that all nodes in your installed {product-title} cluster can access these DNS records.
+The `api.{asterisk}` and `api-int.{asterisk}` DNS records relate to control plane machines, so you must ensure that all nodes in your installed {product-title} cluster can access these DNS records.
```

Comment:
This change shouldn't affect the rendered output of this line (`api.*` and `api-int.*`), just wanted to replace the awkward passthrough method of displaying `*` with a simple asterisk attribute. So there should be no change in the preview compared to the live doc here.